lib/coarse_op.cuh

```diff
@@ -910,6 +715,8 @@ namespace quda {
       switch (type) {
       case COMPUTE_VUV:
         Y_atomic.backup();
+	X_atomic.backup();
```

Comment:
Oops, this one's on me :) We should backport this to release/1.1.x

Reply:
This wasn't a bug in develop / 1.1.  It was an addition here due to how the `switch / case` statements are now written to avoid compiler warnings from `-Wextra`.